barretenberg/cpp/src/barretenberg/dsl/acir_proofs/honk_contract.hpp

```diff
@@ -1727,9 +1728,9 @@ abstract contract BaseHonkVerifier is IVerifier {
 
     function verify(bytes calldata proof, bytes32[] calldata publicInputs) public view override returns (bool) {
         // Check the received proof is the expected size where each field element is 32 bytes
-        if (proof.length != PROOF_SIZE * 32) {
-            revert ProofLengthWrong();
-        }
+        // if (proof.length != PROOF_SIZE * 32) {
```

Comment:
todo

Reply:
is this todo going to be completed before the PR is merged? if not could you create an issue and document in a comment what needs to be done?

Reply:
this has been reinstated